docs/src/main/asciidoc/functional.adoc

```diff
@@ -292,28 +292,21 @@ And to help with correct dependencies here is the excerpt from POM
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-test</artifactId>
         <scope>test</scope>
-        <exclusions>
-            <exclusion>
-                <groupId>org.junit.vintage</groupId>
-                <artifactId>junit-vintage-engine</artifactId>
-            </exclusion>
-        </exclusions>
     </dependency>
 ----
 
 Or you could write a test for a non-HTTP app using just the `FunctionCatalog`. For example:
 
 [source, java]
 ----
-@RunWith(SpringRunner.class)
```

Comment:
Minor thing, but no longer required as this is a Junit4 annotation and `@SpringBootTest` brings alongs its contemporary replacement `@ExtendWith({SpringExtension.class})`.